### PR TITLE
Added template file and tasks to apply retention policies and Continu…

### PR DESCRIPTION
**Pull request by amitbose327**
Added template file and tasks to apply retention policies and Continuous queries on databases.

This PR should go in with: https://github.com/seomoz/roger/pull/403

I tested this out on my local influxdb using the following ansible command:
ansible-playbook -i ../../roger/ansible/hosts/influx monitoring-backend9.yml --tags="retention_policy" --limit=localmesos02 --user=vagrant --ask-pass

@jordmoz and @ankanm Please review.

### CODE REVIEW

**Comment by jordmoz** [comment]
LGTM

File: ansible/roles/monitoring-influxdb9/tasks/main.yml [hunk 1]
```diff
@@ -87,3 +87,15 @@
...
+- name: Copying retention policies and continuous queries on influx host
+  template: src=influxdb.retention.j2 dest=/tmp/influxdb.retention.sh  
+  tags:
+    - retention_policy
+
+# Applying retention policies and continuous queries on influx database
+- name: Applying retention policies and continuous queries on influx database
+  command: sh /tmp/influxdb.retention.sh
+  tags:
+    - retention_policy
```
Comment (ankanm) [inline ansible/roles/monitoring-influxdb9/tasks/main.yml:104]: How about deleting the copied file?

Comment (amitbose327) [inline ansible/roles/monitoring-influxdb9/tasks/main.yml:104]: Done.

**Comment by ankanm** [comment]
LGTM

**Pull request merged**